docs/http/traffic-policy/index.mdx

```diff
@@ -60,13 +60,13 @@ and actions that should take effect.
 
 ## Behavior
 
-Policy rules are evaluated sequentially in the order they are configured with inbound rules taking effect before the
-upstream server is reached and outbound rules taking effect after the upstream server responds. Whether or not the configured
+Traffic Policy rules are evaluated sequentially in the order they are configured with `on_tcp_connect` and `on_http_request` rules taking effect before the
```

Comment:
We should be consistent with capitalization throughout the docs, e.g. "Traffic Policy" or "traffic policy" but not a combo of both.

Reply:
in a blurb where we have already referenced `Traffic Policy`, is it ok to use `Policy` from then on out? Sometimes it can be a bit clunk to use "TrafficPolicy" multiple times within a few sentences 🤷

Reply:
I would lean towards "Traffic Policy" when referring to the module and use "policy" otherwise. This aligns with very first sentence of the doc: "This module enables you to assign a policy to your endpoints ..."

e.g 
"The Traffic Policy module is the best thing ever"
"Policy rules are evaluated sequentially"
"If no expressions are specified on a policy rule, chaos ensues"